build_system/src/build.rs

```diff
@@ -45,8 +45,16 @@ impl BuildArg {
         println!(
             r#"
 `build` command help:
-
-    --sysroot              : Build with sysroot"#
+    --release              : Build backend in release mode with optimized dependencies without rebuilding sysroot
+    --sysroot              : When used on its own, build backend in dev. mode with optimized dependencies
+                             and sysroot in dev. mode (unoptimized)
+                             When used together with --release, build backend in release mode with optimized dependencies
+                             When used together with --release-sysroot,
+                             build the sysroot in release mode with optimized dependencies instead of in dev. mode
+    --release-sysroot      : When combined with --sysroot, additionally
+                             build the sysroot in release mode with optimized dependencies.
+                             When combined with --release, it has no effect.
```

Comment:
I would either remove this line completely or rephrase to:
```suggestion
                             It has no effect if `--sysroot` is not specified.
```